storage/s3_sdk/s3_gcs_test.go

```diff
@@ -42,7 +42,7 @@ func TestBucketList(t *testing.T) {
 	buf := new(bytes.Buffer)
 	err := listGCSBuckets(buf, googleAccessKeyID, googleAccessKeySecret)
 	if err != nil {
-		t.Errorf("listGCSBuckets: %v", err)
+		t.Errorf("%v", err)
```

Comment:
Please keep these the same.

https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures